Stardrop/Views/MainWindow.axaml.cs

```diff
@@ -2558,5 +2613,32 @@ private async Task DisplayInvalidSMAPIWarning()
 
             await DisplaySettingsWindow();
         }
+
+        private void SetupDownloadCountListener()
+        {
+            var downloadPanel = this.FindControl<DownloadPanel>("DownloadPanel");
+            if (downloadPanel is null)
+            {
+                return;
+            }
+
+            if (downloadPanel.DataContext is not DownloadPanelViewModel panelVM)
+            {
+                return;
+            }
+
+            // Initial value
+            _viewModel.DownloadsButtonText = String.Format(Program.translation.Get("ui.main_window.buttons.downloads.label"), 0);
```

Comment:
A question about handling interpolated localization values like this--is there an established pattern for handling stuff like this, where half the string's value has to be localized, but the other half has to be `String.Format()`-ified? I can think of a few solutions  that might work, but I'd prefer to follow whatever existing convention exists.

Reply:
This follows the convention currently used throughout Stardrop! If there is a better route, then that can be used instead.